src/postgres/tree/PostgreSQLSchemaTreeItem.ts

```diff
@@ -0,0 +1,45 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { Schema, Table } from 'pg-structure';
+import * as _ from 'underscore';
+import * as vscode from 'vscode';
+import { AzureParentTreeItem, ISubscriptionContext } from 'vscode-azureextensionui';
+import { getThemeAgnosticIconPath } from '../../constants';
+import { IPostgresTable, PostgreSQLTableTreeItem } from './PostgreSQLTableTreeItem';
+
+export class PostgreSQLSchemaTreeItem extends AzureParentTreeItem<ISubscriptionContext> {
```

Comment:
it's a little weird that in some cases you use `postgres` (like the context value) and some you use `postgresSQL` (for class names). Can we pick a consistent one to use internally? I guess `postgres` seems easier since it has fewer characters. Would also be fine with `pg` if we want to use the least number of characters. Of course, for any user-visible string, it should be the official "PostgreSQL"

Reply:
Should we have the class names start with PG... as well or is Postgres ok?

Reply:
I don't care if you use `Pg` or `Postgres`, but I would prefer if it was consistent